content/en/blog/_posts/2021-XX-XX-writing-an-operator-for-pod-labels.md

```diff
@@ -0,0 +1,447 @@
+---
+layout: blog
+title: "Writing an Operator for Pod Labels"
+date: 2021-03-XX
+slug: writing-an-operator-for-pod-labels
+---
+
+**Authors**: Arthur Busser, Site Reliability Engineer, Padok
+
+[Operators][what-is-an-operator] are proving to be an excellent solution to
```

Comment:
I'd link to our documentation on the operator pattern:
```suggestion
[Operators](/docs/concepts/extend-kubernetes/operator/) are proving to be an excellent solution to
```

Reply:
Corrected 👍